src/reducers/editor.ts

```diff
@@ -0,0 +1,18 @@
+import { SET_UNSAVEDCHANGES, EditorActionTypes } from 'actions/editor';
+
+const initialState = {
+  isUnsaved: false
+};
+
+function editor(state = initialState, action: EditorActionTypes) {
+  switch (action.type) {
+    case SET_UNSAVEDCHANGES:
+      const { payload } = action;
```

Comment:
Siden staten bare inneholder denne ene variabelen, så er det ikke så viktig å spreade. Men gjør jo ikke noe for så vidt ;)